draft-bormann-cbor-yang-standin.md

```diff
@@ -174,14 +174,31 @@ uses the legacy "`-00:00`" format for time-zone-free date-times)
 * do not have fractional parts of the seconds with a precision that
   cannot be represented in floating-point tag content in a tag 1.
 
-All other `date-and-time` values stay in legacy representation.
-
 Tag 1 uses an integer tag content for all `date-and-time` values
 without fractional seconds and a floating-point tag content for values
 that have fractional seconds given.
 
 Tag 100 {{-date}} can unambiguously stand in for all `date-no-zone` values.
 
+## `ietf-yang-types`: Tag 1001 (Extended Date and Time)
+
+{{Section 3 of -legacy-bis}} defines the following types in `ietf-yang-types`:
+
+YANG type | base type | specification | stand-in
+date-and-time | string | {{-yang-types}} | tag 1001
+date-no-zone | string | {{-legacy-bis}} | tag 1001
+date | string | {{-legacy-bis}} | tag 1001
+time | string | {{-legacy-bis}} | tag 1001
+time-no-zone | string | {{-legacy-bis}} | 1001
+{: title="Legacy representations in ietf-yang-types"}
+
+The tag 1001 {{-extended-time}} can unambigously stand for all the
+aforementioned types of values. Values of type `time` and `time-no-zone`
+are encoded with a date of 1 January 1970.
+
+If the encoder supports tag 1001, it MUST NOT use tags 100 and 1 as stand-ins.
```

Comment:
Why?  Tags 100 and 1 are significantly more compact.

Reply:
Well, I thought I had a reason when I was writing it and then it just stayed there. I'm ok with dropping this now as I don't have a reason now.